server.js

```diff
@@ -0,0 +1,24 @@
+console.log('chuck\'s computer is loading express')
+
+const express = require('express');
+const app = express();
+//const bodyParser = require('body-parser)').urlencode({estended: true});
+const PORT = process.env.PORT || 3000;
+app.use(express.static('/portfolio'));
```

Comment:
Looks like you changed this in #14, but this should be `'./public'`, with all your public JS/CSS/HTML/etc files moved into a `public/` folder. See [labs-6-7-structure](https://github.com/DeltaVCode/cr-301-2017/blob/abc7ba9b416bcd246b326c3464fbb77c04c24c14/07-NodeJS-NPM/labs-6-7-structure.pdf).